src/account.ts

```diff
@@ -66,6 +66,21 @@ export abstract class Account {
         calldata?: StringMap,
         options?: InvokeOptions
     ): Promise<InvokeResponse> {
+        let maxFee: starknet.FeeEstimation;
+        if (options?.maxFee && options?.overhead) {
+            const msg = "Both maxFee and overhead cannot be specified";
+            throw new StarknetPluginError(msg);
+        }
+
+        if (options?.maxFee === undefined || options?.maxFee === null) {
+            let overhead = options?.overhead === undefined || options?.overhead === null ? 0.5 : options?.overhead;
+            overhead = Math.round((1 + overhead) * 100);
+            maxFee = await this.estimateFee(toContract, functionName, calldata, options);
```

Comment:
I was confused for some time because I didn't realize this maxFee was already defined in L69. I think it would be better to call this `const estimatedFee`.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/206/commits/656599a8198b1684cbc7edc1d9b02d2b9d86a2e6